tools/autodeployment/env.staging

```diff
@@ -0,0 +1,11 @@
+# Port on which the server will be listening
```

Comment:
These files exist already in the root https://github.com/Seneca-CDOT/telescope/blob/master/env.staging

Reply:
I thought this is for the env vars for the autodeployment and not telescope itself?

Reply:
Sorry, maybe I'm telling you the wrong thing.  cc @manekenpix